images/build/debian-iptables/buster/Dockerfile

```diff
@@ -18,15 +18,12 @@ FROM ${BASEIMAGE}
 
 # Install iptables and ebtables packages from buster-backports
 ARG IPTABLES_VERSION
-RUN ln -s /bin/rm /usr/sbin/rm \
```

Comment:
is dropping the symlink here intentional?

Reply:
Yes, in new Debian-base images the symlink already exists. Keeping it breaks the build. But as we are moving to create first a new Debian-base image and then the debian-iptables I'll revert everything here but the Debian-base things

Reply:
ack, thanks!